crawler/crawler_mesos.py

```diff
@@ -0,0 +1,231 @@
+    'master/event_queue_messages': Stat("gauge", "master/event_queue_messages"),
+    'master/frameworks_connected': Stat("gauge", "master/frameworks_connected"),
+    'master/frameworks_disconnected': Stat("gauge", "master/frameworks_disconnected"),
+    'master/messages_resource_request': Stat("counter", "master/messages_resource_request"),
+    'master/slaves_connected': Stat("gauge", "master/slaves_connected"),
+    'master/slaves_disconnected': Stat("gauge", "master/slaves_disconnected")
+}
+
+# DICT: Mesos 0.22.0, 0.22.1
+STATS_MESOS_022 = {
+    'master/event_queue_dispatches': Stat("gauge", "master/event_queue_dispatches"),
+    'master/event_queue_http_requests': Stat("gauge", "master/event_queue_http_requests"),
+    'master/event_queue_messages': Stat("gauge", "master/event_queue_messages"),
+    'master/frameworks_connected': Stat("gauge", "master/frameworks_connected"),
+    'master/frameworks_disconnected': Stat("gauge", "master/frameworks_disconnected"),
+    'master/messages_decline_offers': Stat("counter", "master/messages_decline_offers"),
+    'master/messages_resource_request': Stat("counter", "master/messages_resource_request"),
+    'master/slaves_connected': Stat("gauge", "master/slaves_connected"),
+    'master/slaves_disconnected': Stat("gauge", "master/slaves_disconnected"),
+    'master/slave_shutdowns_canceled': Stat("counter", "master/slave_shutdowns_canceled"),
+    'master/slave_shutdowns_scheduled': Stat("counter", "master/slave_shutdowns_scheduled"),
+    'master/task_lost/source_master/reason_slave_removed': Stat("counter", "task_lost/source_master/reason_slave_removed"),
+    'master/tasks_error': Stat("counter", "master/tasks_error")
+}
+
+# FUNCTION: gets the list of stats based on the version of mesos
+def get_stats_string(version):
+    if version == "0.19.0" or version == "0.19.1":
+       stats_cur = dict(STATS_MESOS.items() + STATS_MESOS_019.items())
+    elif version == "0.20.0" or version == "0.20.1":
+       stats_cur = dict(STATS_MESOS.items() + STATS_MESOS_020.items())
+    elif version == "0.21.0" or version == "0.21.1":
+       stats_cur = dict(STATS_MESOS.items() + STATS_MESOS_021.items())
+    elif version == "0.22.0" or version == "0.22.1":
+       stats_cur = dict(STATS_MESOS.items() + STATS_MESOS_022.items())
+    else:
+       stats_cur = dict(STATS_MESOS.items() + STATS_MESOS_022.items())
+
+    return stats_cur
+
+
+def configure_crawler_mesos(inurl):
+    print "inurl"
+    print inurl
+    CONFIGS.append({
+        'mesos_url': inurl[0]
+    })
+
+def fetch_stats(mesos_version):
+    print "connecting to"
+    print CONFIGS[0]['mesos_url']
+    try:
+        result = json.loads(urllib2.urlopen(CONFIGS[0]['mesos_url'], timeout=10).read())
+    except urllib2.URLError, e:
+        print "url error"
+        return None
+    print "mesos_stats" 
+    print result
+    return result
+
+
+def setup_logger(logger_name, logfile='crawler.log', process_id=None):
```

Comment:
Don't have to set it up again. You should be able the crawler.log setup at crawler.py.